### ISSUE TITLE
No data in osm_light_points table

### ISSUE THREAD

**Issue by PetersonGIS** [terranodo/tegola-openseamap#6]
I think there should be data in osm_light_points but don't see any in our Italy extract. I believe the two symbols circled in red, below, are representing points that should be in the osm_light_points table. Note, we _do_ have the data in the light_minor and light_major tables.
![light_points](https://user-images.githubusercontent.com/5266205/35772460-5814450a-08fc-11e8-9d71-fc9ed79a5f11.JPG)

**Comment by erictheise** [comment]
Thanks for the specific example, @PetersonGIS. I've gotten far enough with the planet import to identify 36 tables having no data. The planet file is a year old and so can't be expected to match the quantities at [Seamarks/Seamark Object Usage](https://wiki.openstreetmap.org/wiki/Seamarks/Seamark_Object_Usage) but given the low number or complete absence of tags in the OSM db, I have only found one potential error (bolded below, two_way_route_lines).

| osm_table | num_rows | taginfo says |
| --------- | -------- | ------------ |
| osm_causeway_areas               | 0 | 0 |
| osm_causeway_lines               | 0 | 5 |
| osm_checkpoint_areas             | 0 | This tag does not appear in the OSM database. |
| osm_checkpoint_points            | 0 | This tag does not appear in the OSM database. |
| osm_communication_area           | 0 | This tag does not appear in the OSM database. |
| osm_exceptional_structure_areas  | 0 | This tag does not appear in the OSM database. |
| osm_exceptional_structure_lines  | 0 | This tag does not appear in the OSM database. |
| osm_exceptional_structure_points | 0 | This tag does not appear in the OSM database. |
| osm_gridiron_points              | 0 | 0 |
| osm_hulk_points                  | 0 | 0 |
| osm_inshore_traffic_zone_points  | 0 | 0 |
| osm_light_areas                  | 0 | 0 |
| osm_light_points                 | 0 | 4 |
| osm_military_area_points         | 0 | 0 |
| osm_protected_area_points        | 0 | 0 |
| osm_pylon_areas                  | 0 | 0 |
| osm_radar_line                   | 0 | This tag does not appear in the OSM database. |
| osm_radar_range                  | 0 | This tag does not appear in the OSM database. |
| osm_recommended_route_centreline | 0 | This tag does not appear in the OSM database. |
| osm_recommended_traffic_lane     | 0 | This tag does not appear in the OSM database. |
| osm_retro_reflector              | 0 | This tag does not appear in the OSM database. |
| osm_sand_waves_areas             | 0 | This tag does not appear in the OSM database. |
| osm_sand_waves_points            | 0 | This tag does not appear in the OSM database. |
| osm_seaplane_landing_area_points | 0 | 0 |
| osm_separation_crossing          | 0 | This tag does not appear in the OSM database. |
| osm_spring                       | 0 | This tag does not appear in the OSM database. |
| osm_submarine_transit_lane       | 0 | This tag does not appear in the OSM database. |
| osm_turning_basin_points         | 0 | 0 |
| osm_two_way_route                | 0 | __there are 8 two_way_route_lines; fix for next import__ |
| osm_vegetation_areas             | 0 | This tag does not appear in the OSM database. |
| osm_vegetation_lines             | 0 | This tag does not appear in the OSM database. |
| osm_vegetation_points            | 0 | This tag does not appear in the OSM database. |
| osm_vehicle_transfer_areas       | 0 | This tag does not appear in the OSM database. |
| osm_vehicle_transfer_points      | 0 | This tag does not appear in the OSM database. |
| osm_water_turbulence_areas       | 0 | 0 |
| osm_water_turbulence_lines       | 0 | 1 |

**Comment by erictheise** [comment]
The `two_way_route` import actually appears to be correct and consistent with what I've done elsewhere so I think there simply are none in the planet file we have.

**Comment by PetersonGIS** [comment]
Great info.
From that then, it now seems pretty apparent that the "missing" data for the light symbology should be in the light_major and light_minor tables **but it is not**.
Looking here: https://wiki.openstreetmap.org/wiki/Tag:seamark:type=light_minor we see that anything tagged with light_minor would usually also have at least a colour (and probably an orientation) associated with it, but our tables are empty for these fields even though the fields exist. The style json will use the colour and orientation fields to assign the appropriately colored and oriented teardrop shape to the star shape.

**Comment by PetersonGIS** [comment]
^so in summary we need to figure out how to get data into the light_minor and light_major tables. Our tables have the right fields, but no data in those fields.

**Comment by erictheise** [comment]
Agreed, I've been digging into this, which will also impact #7, since last night.
